app/lib/activitypub/tag_manager.rb

```diff
@@ -35,6 +35,8 @@ def uri_for(target)
       account_status_url(target.account, target)
     when :emoji
       emoji_url(target)
+    when :vote
+      [uri_for(target.account), '#votes/', target.id].join
```

Comment:
Well, if they are going to be stored, might generate a real path for them?

Reply:
we could, but I'm not sure if that is useful, much like follows

Reply:
I don't disagree, but the follows &co are not in this method :confused: Perhaps you could move them here then?

Reply:
Changed it to use the same logic as `Follow` and cie